CPAC/pipeline/cpac_group_runner.py

```diff
@@ -41,18 +40,19 @@ def load_config_yml(config_file, individual=False):
 
         config_dict = yaml.safe_load(open(config_path, 'r'))
 
-        config = Configuration(config_dict)
+        #config = Configuration(config_dict)
+        config = config_dict
 
     except Exception as e:
         err = "\n\n[!] CPAC says: Could not load or read the configuration " \
               "YAML file:\n%s\nDetails: %s\n\n" % (config_file, e)
         raise Exception(err)
 
     if individual:
-        config.pipeline_setup['log_directory']['path'] = os.path.abspath(config.pipeline_setup['log_directory']['path'])
-        config.pipeline_setup['working_directory']['path'] = os.path.abspath(config.pipeline_setup['working_directory']['path'])
-        config.pipeline_setup['output_directory']['path'] = os.path.abspath(config.pipeline_setup['output_directory']['path'])
-        config.pipeline_setup['crash_log_directory']['path'] = os.path.abspath(config.pipeline_setup['crash_log_directory']['path'])
+        config.logDirectory = os.path.abspath(config.logDirectory)
+        config.workingDirectory = os.path.abspath(config.workingDirectory)
+        config.outputDirectory = os.path.abspath(config.outputDirectory)
+        config.crashLogDirectory = os.path.abspath(config.crashLogDirectory)
```

Comment:
Are these supposed to be different than like https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L1176-L1177 https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L1337-L1340 ?

Reply:
Yes they are, completely missed them. Thank you!

Reply:
Looking back at this, I don't see how it would work since it would just replace the same object in the config dictionary.

Reply:
I'm asking about what looks like https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L52-L55 would take this structure
```YAML
logDirectory:
workingDirectory:
outputDirectory:
crashLogDirectory:
```
and update the paths to absolute paths, but other sections like https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L1176-L1177 https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/cpac_group_runner.py#L1337-L1340 look like they'd use a structure like
```YAML
pipeline_setup:
  log_directory:
    path:
  output_directory:
    source_outputs_path:
    output_path:
  working_directory:
    path:
```

Reply:
Ah I see what you mean, yes I'll change them for the config tree.